src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/DefaultJsonTypeInfoResolver.Helpers.cs

```diff
@@ -7,30 +7,38 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 using System.Text.Json.Reflection;
+using System.Threading;
 
 namespace System.Text.Json.Serialization.Metadata
 {
     public partial class DefaultJsonTypeInfoResolver
     {
         internal static MemberAccessor MemberAccessor
         {
+            [RequiresUnreferencedCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
             [RequiresDynamicCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
```

Comment:
Probably doesn't matter, but should this be ifdef'd to NETCOREAPP | NETFRAMEWORK?

Reply:
They're being used heavily inside the library, so that would require a lot of ifdef'ing. I'm personally fine with those showing up redundantly in ns2.0 builds.